### ISSUE TITLE
SSH Banner

### ISSUE THREAD

**Issue by cognitivegears** [NuSkooler/enigma-bbs#429]
**Is your feature request related to a problem? Please describe.**

N/A

**Describe the solution you'd like**

From the source, in `ssh.js` there is a comment:

```js
    // Note that sending 'banner' breaks at least EtherTerm!
```

I wonder if this is still true. With the upgrade on Enigma 1/2 side to the ssh library, plus EtherTerm has had a major release since this comment was made. I can do some testing with EtherTerm if you are open to trying this out. Personally I've tried banner with a few (modern) clients with good effect.

Otherwise, `SSHPMPT.ASC` is limited to only after a failed authentication attempt when trying `keyboard-interactive`.

**Describe alternatives you've considered**

`greeting` is usually ignored.

**Additional context**
N/A

**Comment by NuSkooler** [comment]
@cognitivegears any update on this one? If we can get a column added to the table in the compat doc (https://docs.google.com/spreadsheets/d/1j59keJZYtulyt6XRn0vYniJ6HLiVuQzuER5N0nYZYEo/edit?usp=sharing) that would help.

**Comment by cognitivegears** [comment]
Sorta :) I did try EtherTerm, but had some trouble getting it to compile on Linux. I haven't yet tried the Windows binaries yet under a VM or anything. In the meantime though, I realized that we don't really need the banner...

My original thought was that it's a shame that there is only a single list of new user commands and that otherwise you had to have a username/password before ever seeing any art or information. For example, the example BBS that I setup has an "about" screen to give people some info before joining.

However, thinking about it I don't think a banner really solves this problem anyway. Plus, I found a better way to do it. I haven't committed a PR for it yet, but with a tweak to add a second "anonymousUsers" list, as well as a change to populate the user id from an unauthenticated user into the user name MCI, it's possible to create ssh login screens that work for unauthenticated users. With the MCI change, menus can be then configured for any type of unauthenticated access needed over ssh. This is much more general anyway besides the specific idea of displaying a banner, so if you agree I think we should close this issue.
